packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -762,9 +822,101 @@ describe('cluster', () => {
     cluster.addRotationSingleUser();
 
     // THEN
-    expect(() => cluster.addRotationSingleUser()).toThrow(/A single user rotation was already added to this cluster/);
+    expect(() => cluster.addRotationSingleUser()).toThrow(/A single user rotation for master secret was already added to this cluster/);
```

Comment:
Let's re-format this while we're in the area:

```suggestion
    expect(() => {
      cluster.addRotationSingleUser();
    }).toThrow(/A single user rotation for master secret was already added to this cluster/);
```